src/components/bucket-utils.js

```diff
@@ -22,18 +22,23 @@ export const withRequesterPaysHandler = _.curry((handler, fn) => async (...args)
 export const requesterPaysWrapper = ({ onDismiss }) => WrappedComponent => {
   return Utils.forwardRefWithName('requesterPaysWrapper', (props, ref) => {
     const [showModal, setShowModal] = useState(false)
+    const [onSuccess, setOnSuccess] = useState(() => {})
```

Comment:
Can we make this more specific than onSuccess? Like maybe `onRequesterPaysProjectSelected`?

Also, could we make this a prop passed into the wrapper? It feels kind of strange that it needs to be set on the error for `onSuccess` to be called.